tests/test_parameters_definition.py

```diff
@@ -0,0 +1,161 @@
+import numpy as np
+import pytest
+import pyfar as pf
+import pyrato as ra
+from pyrato.parameters import definition
+import re
+
+
+def make_edc_from_energy(energy, sampling_rate=1000):
```

Comment:
A copy of this function is already in the clarity pull. Lets discuss during the weekly how we want to handle this. In pyfar we have `pyfar.testing`, maybe we can do this here as well.

Reply:
If that is the case, we could think about joining the tests in `test_parameters.py` instead of having a separate file for each parameter. In this case the helper function could maybe also stay there.

Reply:
I opened an issue to keep track of this and would suggest to not fix it here or discuss in the weekly before fixing: #81